Makefile

```
@@ -34,6 +34,11 @@ fix-format:
 test-evm:
 	cd evm && forge test -vvv
 
+.PHONY: anchor-lint
+anchor-lint:
+	cd solana && make anchor-lint
```

nit: 
```suggestion
	make -C solana anchor-lint
```
is a little more idiomatic. As an aside, I'm not sure this top-level makefile is all that useful. I think it would be better to just have `evm/Makefile` and `solana/Makefile`

---

cool, I'll update make the idiomatic update in a second PR if that's OK with you, just so we can get the actual logic bits merged.

I think that's my preference too, but while we have a top-level Makefile I figure it should probably actually do something for Solana. The main goal anyway is just to create a target that captures the actual GitHub CI commands so that they can be run locally in a convenient way.